workflow/rules/utils.smk

```diff
@@ -14,3 +18,42 @@ def make_paths_absolute(obj, base_path):
 			return new_file
 	return obj
 
+def _get_biosample_model_dir(biosample):
```

Comment:
@mayasheth here's where we map the biosample row to the model. We'll update this function as more models get supported